metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDBTest.java

```diff
@@ -18,6 +18,7 @@
 
 package uk.ac.ebi.ampt2d.metadata.importer.database;
 
+import org.junit.Assert;
```

Comment:
Can be removed (org.junit.Assert.assertEquals is included)

Reply:
Thanks, fixed in https://github.com/EBIvariation/metadata-ws/pull/55/commits/0e5f9b50a3fc908dd654a8d64e7aac5919050f3a